pytximport/core/_tximport.py

```diff
@@ -553,7 +576,17 @@ def tximport(
 
         log(25, f"Saving the gene-level expression to: {output_path}.")
 
-        if output_format == "h5ad":
+        if output_format.lower() == "summarizedexperiment":
```

Comment:
`.lower()` deviates from the implementation for the other formats here. Should either apply to all or none, for now I would prefer not to use .lower() since touching the part for the other formats is not directly related to this PR and the general change should be discussed and implemented in a different PR. Additionally, since support for SummarizedExperiment is only optional, it should not come first in the if/elif block.